docs/UserGuide.adoc

```diff
@@ -400,29 +399,59 @@ To match all the patients in ORGANice, do the following:
 image::Patient Matches Displayed.png[width=790]
 +
 
+Following this, you can sort the matches based on different categories to deduce which patient to prioritize.
++
+Refer to refer to <<Sorting matches: `sort`>> for more information.
 
 === Sorting matches: `sort`
-Users can specify how they want the patient-donor pairs to be sorted. +
-Sorting is done by passing in optional parameters to the `match` command. +
+After matching patients and donors, you can sort the list of matches to determine which patient to prioritize for organ transplant.
 
-Matches can be sorted according to three categories:
+**The sort command can only be used after a match command** to sort the resulting list of patient-donor pairs generated.
+There are three categories by which you can sort the matches, namely:
 
-1. Success rate of match
-2. Priority of patient
-3. Date of expiry of donors' organs
+1. Sorting based on donor's organ expiry date
```

Comment:
i prefer bullet points though cause order here doesnt matter. What do you think?

Reply:
changed.